### ISSUE TITLE
ERD tab should be removed from the Database Editor

### ISSUE THREAD

**Issue by kseniiaguzeeva** [dbeaver/dbeaver#11006]
#### System information: 
- Operating system (distribution) and version Win10
- DBeaver version 7.3.3 EA

#### Connection specification:
- Database name and version Informix

#### Steps to reproduce, if exist:
Open a database in Object Editor

Expected: Properties tab is only available
Actual result: Empty ER Diagram tab is displayed too

**Comment by serge-rider** [comment]
Removed in some previous fixes

**Comment by uslss** [comment]
still actual for Informix in EE. 

![image](https://user-images.githubusercontent.com/31996417/117782506-5d71df80-b24a-11eb-8fd9-233afee225ee.png)

**Comment by serge-rider** [comment]
Unfortunately we can't fix this for pure Generic driver (drivers which use Generic metadata structure, like Informix.) 
Because we don't know database structure before we start loading it. So ERD tab is visible on catalogs which weren't initialized yet.

This may be fixed with fully-functional Informix extension but it should be done in a separate ticket.
